.github/workflows/docker-builder-devcontainer.yml

```diff
@@ -53,9 +54,9 @@ env:
 jobs:
   build_dockerfile_bazel_base:
     if: |
-      github.event_name == 'push' &&
+      (github.event_name == 'push' &&
       github.ref_name == 'master' &&
-      github.repository_owner == 'magma'
+      github.repository_owner == 'magma' || github.event_name == 'workflow_dispatch')
```

Comment:
can you please still enforce that just `ref_name == master` and `repository_owner == magma`, even for workflow_dispatch?

Reply:
@maxhbr  I have modified the PR as per your suggestion.